lesson_13/roman_alex/lesson_13_roman_alex.py

```diff
@@ -0,0 +1,59 @@
+from math import sqrt
+
+class PerimeterMixin:
+
+    def perimeter(self):
+        return f'Perimeter is {sum(self.sides)}'
+
+
+class Polygons(PerimeterMixin):
```

Comment:
The PerimeterMixin is correct, but we want to have polygons that are able to use the perimeter method and some polygons that don't need to have this method. In the challenge is specified that only the Triangle class should have the perimeter method. If we use it like this all the shapes that inherit from Polygon will have the method perimeter.

Where can we move the Mixin to have the perimeter only on the Triangle class?

Reply:
Code updated...